src/presentational/DialogScreen-Download.jsx

```diff
@@ -5,6 +5,7 @@ import {saveAs} from 'browser-filesaver';
 export default class DialogScreen_DownloadCSV extends DialogScreen {
   constructor(props) {
     super(props);
```

Comment:
I'm not entirely sure if I should remove this super() call, because on FF & Chrome it means that `this.closeMe = this.closeMe.bind(this)` will trigger twice. I mean, it _works_, but it's such a hack just to get things working for IE10.